Storybook/Pages/GameView/AnimationPage/AnimationPageScene2.swift

```diff
@@ -0,0 +1,164 @@
+//
+//  AnimationPageScene.swift
+//  Storybook
+//
+//  Created by Adam Ibnu fiadi on 10/10/22.
+//
+
+import Foundation
+import SpriteKit
+import UIKit
+
+class AnimationPageScene2: GameScene {
```

Comment:
Don't get used to naming classes like 'AnimationPageScene2' etc, please define it more clearly so another programmer can understand what this scene is for. @adamfiadi7007